test/test-utils/gameManager.ts

```diff
@@ -502,7 +502,7 @@ export class GameManager {
    */
   selectPartyPokemon(slot: number, inPhase = "SwitchPhase"): void {
     this.onNextPrompt(inPhase, UiMode.PARTY, () => {
-      const partyHandler = this.scene.ui.getHandler() as PartyUiHandler;
+      const partyHandler = this.scene.ui.getCurrentHandler() as PartyUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
      const partyHandler = this.scene.ui.getCurrentHandler<PartyUiHandler>();
```